workspace/manager.py

```
@@ -240,6 +240,105 @@ def create_workspace(
+
+        For non-GitHub hosts this check is not guaranteed and will return a
+        not-supported error.
+        """
+        if not repo_url or not path:
+            return {"ok": False, "error": "missing_repo_or_path"}
+        # Try GitHub API when possible
+        try:
+            if repo_url.startswith("https://github.com/") or repo_url.startswith("http://github.com/"):
+                # Parse owner/repo from https URL
+                stripped = repo_url.rstrip("/ ")
+                if stripped.endswith(".git"): stripped = stripped[:-4]
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)